modules/nf-core/narfmap/align/tests/main.nf.test

```diff
@@ -84,10 +86,12 @@ nextflow_process {
             assertAll (
                 { assert process.success },
                 { assert snapshot(
+                    process.out.bam,
+                    process.out.versions,
                     file(process.out.bam[0][1]).name,
```

Comment:
I think you can remove this line as well,since you're snapshotting process.out.bam already